tests/conftest.py

```diff
@@ -2767,3 +2767,88 @@ def ns_resources_and_connections_cleanup():
     request.addfinalizer(ns_resources_and_connections_cleanup)
 
     return _create_ns_resources
+
+
+@pytest.fixture()
+def snapshot_restore_factory(request):
+    """
+    Snapshot restore factory. Calling this fixture creates new PVC out of the
+    specified VolumeSnapshot.
+
+    """
+    instances = []
+
+    def factory(
+        snapshot_obj,
+        storageclass=None,
+        size=None,
+        volume_mode=None,
+        restore_pvc_yaml=None,
+        access_mode=constants.ACCESS_MODE_RWO,
+        status=constants.STATUS_BOUND
+    ):
+        """
+        Args:
+            snapshot_obj (OCS): OCS instance of kind VolumeSnapshot which has
+                to be restored to new PVC
+            storageclass (str): Name of storageclass
+            size (str): Size of PVC being created. eg: 5Gi. Ideally, this
+                should be same as the restore size of snapshot. Adding this
+                parameter to consider negative test scenarios.
+            volume_mode (str): Volume mode for PVC. This should match the
+                volume mode of parent PVC.
+            restore_pvc_yaml (str): The location of pvc-restore.yaml
+            access_mode (str): This decides the access mode to be used for the
+                PVC. ReadWriteOnce is default.
+            status (str): If provided then factory waits for the PVC to reach
+                desired state.
+
+        Returns:
+            OCS: OCS instance of kind VolumeSnapshot
+
+        """
+        from ocs_ci.ocs.resources import pvc
```

Comment:
can we put the import at the top of the file?

Reply:
Removed the import